### PR TITLE
feat(theme): add accent color selection

### PR DESCRIPTION
**Pull request by jpnurmi**
| Before | After |
|---|---|
| image | image |

[Screencast from 2023-06-28 15-54-56.webm](https://github.com/canonical/ubuntu-desktop-installer/assets/140617/29c31b1b-15aa-485c-802d-92c352f64776)

### CODE REVIEW

**Review by d-loose — APPROVED** [review]
:tada:

File: packages/ubuntu_desktop_installer/lib/pages/theme/theme_page.dart [hunk 1]
```diff
@@ -29,30 +31,51 @@ class ThemePage extends ConsumerWidget {
...
+          ),
+          if (yaru != null)
+            Center(
+              child: SingleChildScrollView(
+                padding: const EdgeInsets.only(top: kWizardSpacing * 2),
+                scrollDirection: Axis.horizontal,
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    for (final variant in YaruVariant.values.take(10))
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/pages/theme/theme_page.dart:68]:
```suggestion
                    for (final variant in YaruVariant.accents)
```
- https://github.com/ubuntu/yaru.dart/pull/359

**Review by d-loose — APPROVED** [review]
LGTM :+1:

Nevermind, I already had a look at this one :D

**Comment by jpnurmi** [comment]
@elioqoshi I'm not sure if we need a formal UX review at this point because the page will be moved from the installer to the welcome wizard which you get to design but I thought I'd let you chime in anyway. :)

**Comment by elioqoshi** [comment]
Thanks @jpnurmi this is a great improvement! I will probably redesign this screen a tiny bit once the time comes indeed (names for the colors e.g) but this is great already.

**Pull request merged**